src/handlers/hello-world.ts

```diff
@@ -0,0 +1,59 @@
+import { Context } from "../types";
+
+/**
+ * NOTICE: Remove this file or use it as a template for your own plugins.
+ *
+ * This encapsulates the logic for a plugin if the only thing it does is say "Hello, world!".
+ *
+ * Try it out by running your local kernel worker and running the `yarn worker` command.
+ * Comment on an issue in a repository where your GitHub App is installed and see the magic happen!
+ *
+ * Logger examples are provided to show how to log different types of data.
+ */
+export async function helloWorld(context: Context) {
+  const {
+    logger,
+    payload,
+    octokit,
+    config: { configurableResponse },
+  } = context;
+
+  const sender = payload.comment.user?.login;
+  const repo = payload.repository.name;
+  const issueNumber = payload.issue.number;
+  const owner = payload.repository.owner.login;
+  const body = payload.comment.body;
+
+  if (!body.match(/hello/i)) {
+    logger.error(`Invalid use of slash command, use "/hello".`, { body });
+    return;
+  }
+
+  logger.info("Hello, world!");
+  logger.debug(`Executing helloWorld:`, { sender, repo, issueNumber, owner });
+
+  try {
+    await octokit.issues.createComment({
+      owner: payload.repository.owner.login,
+      repo: payload.repository.name,
+      issue_number: payload.issue.number,
+      body: configurableResponse,
+    });
+  } catch (error) {
+    /**
+     * logger.fatal should not be used in 9/10 cases. Use logger.error instead.
```

Comment:
It would be good to add 1/10 case when logger.fatal should be applied.

Reply:
Just leaving a note and resolving.

Reply:
as far as plugin devs are concerned technically it's 10/10 times and should be reserved for when ubiquity is the cause of the issue or at least it is our responsibility to fix it (my understanding)

I did first write a comment saying never to use it but it felt wrong so I wasn't sure what to do. Any suggestions knowing the above?

Reply:
I think we should only use fatal from within the kernel and core features like permit generation. 

My vision was to make a triage telegram bot/group to immediately jump on any fatal issues that affect every user.